x/crosschain/keeper/initiate_outbound.go

```diff
@@ -38,6 +38,6 @@ func (k Keeper) InitiateOutbound(ctx sdk.Context, config InitiateOutboundConfig)
 		)
 	}
 
-	config.CCTX.SetPendingOutbound("")
+	config.CCTX.SetPendingOutbound(types.StatusMessages{StatusMessage: "initiating outbound"})
```

Comment:
```suggestion
	config.CCTX.SetPendingOutbound(types.StatusMessages{StatusMessage: "processing outbound"})
```

Reply:
I choose to use initiate , as that's the function called right after this line